provision.sh

```diff
@@ -0,0 +1,2 @@
+# Switch to /srv/huboard on login
+echo "cd /srv/huboard" >> /home/vagrant/.bashrc
```

Comment:
Lets pair together on moving this to puppet, its probably best to keep our provisioning unified.